server/build.gradle

```diff
@@ -23,6 +25,17 @@ test {
     jvmArgs '-XX:+HeapDumpOnOutOfMemoryError'
 }
 
+// substitut the version & release in the version.properties used by the status resource at runtime
+processResources {
+    from('src/main/resources') {
+        include 'version.properties'
+        filter(ReplaceTokens, tokens: [
```

Comment:
could use `expand` instead of `filter`, and then we don't have to change the tests or the properties templates.

Reply:
Expand is cleaner than filter so I switched the usage. Unfortunately we still need an update to the spec test for clarity.